codecs/src/test/scala/org/virtuslab/akkasaferserializer/BorerSerializationTestKit.scala

```diff
@@ -0,0 +1,14 @@
+package org.virtuslab.akkasaferserializer
+
+import io.bullet.borer.derivation.MapBasedCodecs.deriveAllCodecs
+import io.bullet.borer.{Cbor, Codec}
+import org.scalatest.Assertion
+import org.scalatest.matchers.should
+
+trait BorerSerializationTestKit extends should.Matchers {
+  protected def roundTrip[T: Codec](message: T): Assertion = {
```

Comment:
Nit: see scaladoc of `io.bullet.borer.Codec`, they (quite reasonably) recommend `: Encoder : Decoder` instead of `: Codec`